internal/cli/clients/install_client.go

```diff
@@ -159,6 +159,47 @@ func (c *InstallClient) Install(cmd *cobra.Command) error {
 
 	installationWg.Wait()
 
+	// With all deployments done it is now the time to perform
+	// some post-install actions:
+	//
+	// - For a local deployment (using a self-signed cert) get the
+	//   CA cert and save it into the config. The regular client
+	//   will then extend the Cert pool with the same, so that it
+	//   can cerify the server cert.
+
+	if strings.Contains(domain.Value.(string), "omg.howdoi.website") {
+		// Note 1:
+		// We are waiting here for the secret, instead of
+		// simply Get'ting it, because we cannot be sure that
+		// it was already created by cert-manager, from the
+		// cert request issued by the epinio deployment.  This
+		// is especially true when the epinio deployment does
+		// not wait for the server to be up. As it happens
+		// when a dev `ep install` is done (do't wait for
+		// deployment + skip default org).
+		//
+		// Note 2:
+		// The secret's name is (namespace + '-tls'), see the
+		// `auth.createCertificate` template for the created
```

Comment:
[nit] I think this part of the comment is obvious from the lines below. Remove the first part about the name of the secret?

Reply:
Ok.

Reply:
Done, commit d9f2b91cd6fa548012900ff03d3b90a653c3ebb7